### DISCUSSION TITLE
Is it possible to modify the creation of the Reaction Network?

### DISCUSSION THREAD

**Discussion by BiocharModeling** [deepmodeling/reacnetgenerator#2119] [Q&A] [answered]
Hello!
I wanted to know if there is any way to modify the reaction network; for example, if I want to see the entire network of all the compounds involved in the process and not just the 20 main compounds, is it possible to do this?

Thank you a lot!

**Comment by njzjz** [comment]
It can be controlled by `--maxspecies` from the command line.

**Reply by BiocharModeling** [reply]
Thank you for your reply!
Another question would be if it is possible to extract the information of the network to manipulate how it is plotted; for example, if I wanted to plot the network in another way/software.

**Reply by njzjz** [reply, marked as answer]
Please follow https://reacnetgenerator.njzjz.win/guide/report
